src/views/CompleteRegistrationComponent.tsx

```diff
@@ -278,9 +287,9 @@ export default function CompleteRegistrationComponent() {
             <Button
               variant="contained"
               sx={{ alignSelf: 'center' }}
-              onClick={() => window.location.assign(ENV.URL_FE.LANDING)}
+              onClick={() => window.location.assign(buildAssistanceURI(assistanceEmail))}
```

Comment:
read the assistanceEmail from ENV object

Reply:
see: https://github.com/pagopa/selfcare-dashboard-frontend/blob/main/src/components/Layout/Layout.tsx#L39